### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by nini22P**
Update dependencies.
Support asia game images.
And more...

### CODE REVIEW

**Comment by IllusionMan1212** [comment]
Hey, thanks for taking an interest in contributing to the project. I will review this PR in 8-10 hours

File: app/build.gradle [hunk 1]
```diff
@@ -1,54 +1,40 @@
...
-    compileSdkVersion 30
-    buildToolsVersion "29.0.3"
+    namespace 'com.illusionware.npsbrowser'
+    compileSdkVersion 33
 
     defaultConfig {
         applicationId "com.illusionware.npsbrowser"
-        minSdkVersion 23
-        targetSdkVersion 30
+        minSdkVersion 24
```
Comment (IllusionMan1212) [inline app/build.gradle]: any reason for requiring 24 as the minimum sdk version?

File: app/build.gradle [hunk 2]
```diff
@@ -1,54 +1,40 @@
...
-        enabled = true
-    }
-
-    signingConfigs {
-        release {
-            storeFile file(keystoreProperties['storeFile'])
-            storePassword keystoreProperties['storePassword']
-            keyAlias = keystoreProperties['keyAlias']
-            keyPassword keystoreProperties['keyPassword']
-        }
```
Comment (IllusionMan1212) [inline app/build.gradle]: I would like to keep the signing config so I could make release builds with my keystore

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.illusionware.npsbrowser">
```
Comment (IllusionMan1212) [inline app/src/main/AndroidManifest.xml]: Can you explain why you removed the `xmlns:tools` and `package` attributes? I'm not very familiar with android development

**Review by IllusionMan1212 — COMMENTED** [review]
great work. I have a few comments. once those are addressed I can go ahead with merging this.

File: app/build.gradle (hunk 1 above)
Comment (nini22P) [reply app/build.gradle]: Vita3k minimum sdk is 24

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (nini22P) [reply app/src/main/AndroidManifest.xml]: `xmlns:tools` was deleted by mistake, `package` was changed to `namespace` and moved to `app/build.gradle`

**Comment by nini22P** [comment]
Thank you for reviewing this PR, waiting for some time to process

File: app/build.gradle (hunk 2 above)
Comment (nini22P) [reply app/build.gradle]: It will be restored

File: app/build.gradle (hunk 1 above)
Comment (IllusionMan1212) [reply app/build.gradle]: This is a standalone app and is not related to Vita3K in any way. 23 minimum is fine if there are no technical obstacles for it

**Comment by nini22P** [comment]
Completed

**Review by IllusionMan1212 — APPROVED** [review]
thank you!

**Pull request merged**